frontend/src/ui/Blocks/Series.tsx

```diff
@@ -208,6 +232,32 @@ const VideoGrid: React.FC<React.PropsWithChildren> = ({ children }) => (
     </div>
 );
 
+const UpcomingEventsGrid: React.FC<React.PropsWithChildren> = ({ children }) => {
+    const { t } = useTranslation();
+
+    return <details css={{
```

Comment:
When the opening tag after a `return` spans more than one line, I prefer to add `()` like this:

```javascript
return (
    <details ....>
);
```

That's because in your current version, the `}}>` end of the opening tag now has the same indentation level as the `return`. In basically all other cases, a line with the same indentation level indicates that the statement (`return` in this case) ends there. But here it goes on.

Julian usually did it like you did and was mildly annoyed by my preference regarding this :D Maybe I need to ignore this, if everyone likes to write it differently :shrug: